petab/v2/core.py

```diff
@@ -2339,6 +2360,71 @@ def get_output_parameters(
 
         return output_parameters
 
+    def has_timepoint_specific_overrides(
+        self,
+        ignore_scalar_numeric_noise_parameters: bool = False,
+        ignore_scalar_numeric_observable_parameters: bool = False,
```

Comment:
Why is scalar/non-scalar problematic? e.g. my overrides could look like `3` for a single observable parameter, or `3;1.2` for two observable parameters. Why is the `3;1.2` not possible compared to `3`?